src/dateformats.js

```diff
@@ -1,8 +1,10 @@
 define([
     'jquery',
-    'tpl!vellum/templates/date_format_menu'
+    'underscore',
+    'vellum/templates/date_format_menu.html'
 ], function(
     $,
+    _,
```

Comment:
I don't see where `_` is referenced in this file. Is it a template vestige?

Reply:
21c46756

Reply:
21c46756